build-ova.sh

```diff
@@ -69,6 +69,11 @@ function packer_logging() {
 
 # Invokes kubernetes image builder for the corresponding OS target
 function trigger_image_builder() {
+    # Check out image builder with specific commit for the
+    # corresponding k8s version
+    cd ${image_builder_root}
+    git checkout ${IMAGE_BUILDER_COMMIT_ID}
```

Comment:
How is this variable passed to the container during the docker run command?

Reply:
The makefile variables are created as environment variables